Targets of a Transition is not removed while using removeTarget (int targetId)

Consider the following code:

     Transition transition = ...;

     transition.addTarget(R.id.my_view);

     ...

     transition.removeTarget(R.id.my_view);


It is apparent from the [documentation](https://github.com/andkulikov/transitions-everywhere/blob/master/library/src/main/java/android/transitions/everywhere/Transition.java#L1061) that the call to `Transition.removeTarget(int)` should remove the view with id `R.id.my_view` being included in further transitions. But instead the following error will be shown:

     Caused by: java.lang.IndexOutOfBoundsException: Invalid index <some-random-int>, size is 1
            at java.util.ArrayList.throwIndexOutOfBoundsException(ArrayList.java:255)
            at java.util.ArrayList.remove(ArrayList.java:403)

this is because the implementation is calling ArrayList.remove(int) instead of ArrayList.remove(Object).

*PS: the upstream bug is at http://b.android.com/167061 or [here](https://github.com/android/platform_frameworks_base/issues/62)*

--- a/library/src/main/java/android/transitions/everywhere/Transition.java
+++ b/library/src/main/java/android/transitions/everywhere/Transition.java
@@ -1070,7 +1070,7 @@ public abstract class Transition implements Cloneable {
      */
     public Transition removeTarget(int targetId) {
         if (targetId > 0) {
-            mTargetIds.remove(targetId);
+            mTargetIds.remove((Integer) targetId);
         }
         return this;
     }

